src/web/assets/cp/src/js/PreviewFileModal.js

```diff
@@ -60,8 +64,21 @@ Craft.PreviewFileModal = Garnish.Modal.extend(
 
         this.$shade.velocity('stop');
         this.$shade.show().css('opacity', 1);
+
+        Garnish.setFocusWithin(this.$container);
       }
 
+      // Add bumper elements to maintain focus trap
+      this.$bumperButtonStart = Craft.ui.createButton({
+        html: Craft.t('app', 'Close preview'),
```

Comment:
We have an existing `Close Preview` translation (capital `P`), and `Close`, but no `Close preview` yet. Let’s go with one of the existing ones here.